Apps/WebClient/test/functional/e2e/cypress/integration/healthinsights/insights.js

```diff
@@ -1,15 +1,15 @@
 const { AuthMethod } = require("../../support/constants")
 
 describe('Health Insights', () => {
-    beforeEach(() => {
+    before(() => {
         cy.login(Cypress.env('keycloak.username'), 
                  Cypress.env('keycloak.password'), 
                  AuthMethod.KeyCloak)
+        cy.get('#covid-modal___BV_modal_header_ > .close').click()
```

Comment:
Doesn’t the modal have a dataid?  Also, why not stub the config to turn off the modal?  We should also think about making a story to preserve the route path on login.

Reply:
doesn't seem we can add dataid to the close button, but I will check it out. stub the config is interesting idea. Can you expand a bit on how to do so?

Reply:
I believe you're right but you should be able to get the modal and then fund the button within it.

Reply:
sounds good, I will use this for other tests. for this modal, we leverage the configuration to disable them wherever possible.